### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by sjp38**
This pull request fixes some trivial typos.

### CODE REVIEW

File: Documentation/teaching/info/contributing.rst [hunk 1]
```diff
@@ -70,7 +70,7 @@ Forking the repository
    .. code-block:: bash
 
      $ mkdir -p ~/so2
-     $ git clone git@github.com:linux-kernel-labs/linux.git ~/so2/linux
```
Comment (tavip) [inline Documentation/teaching/info/contributing.rst:73]: This seems like a reasonable change to me,  @dbaluta  @razvand what do you think?

**Review by tavip — COMMENTED** [review]
Looks good to me, thank you very much for your contributions @sjp38 !

**Review by tavip — APPROVED** [review]

File: Documentation/teaching/info/contributing.rst (hunk 1 above)
Comment (sjp38) [reply Documentation/teaching/info/contributing.rst:73]: And, I forgot updating the previous line to `mkdir -p ~/src/`.  Will update soon.

**Review by dbaluta — APPROVED** [review]

**Comment by dbaluta** [comment]
@sjp38 Looks good to me. Thanks for your contribution!

**Pull request merged**